.travis.yml

```diff
@@ -21,6 +21,8 @@ matrix:
       - docker
   - env: TEST_NODE_VERSION=6 LINT=1 COVER=1
   - env: TEST_NODE_VERSION=4
+  - env: TEST_NODE_VERSION=--lts
+  allow_failures:
```

Comment:
what is this for, and it is indented correctly?

Reply:
The latest version of node seems to be unstable at this time, so I added the lts version to the build and move the latest to the 'allow_failures' sections which will still build it but won't fail the entire build if it fails

Reply:
makes total sense, thanks